### PR TITLE
feat: added missing MITRE ATT&CK tags

### PR DESCRIPTION
**Pull request by seishinsha**
**What type of PR is this?**

/kind feature

/kind rule-update

**Any specific area of the project related to this PR?**

/area rules

**What this PR does / why we need it**:

This PR includes the `MITRE ATT&CK` tags for Enterprise and for Containers. Tagging rules according to `MITRE ATT&CK` matrix is an important method of classification and will be useful for any user. 

**Which issue(s) this PR fixes**:

NONE

**Special notes for your reviewer**:

NONE

**Does this PR introduce a user-facing change?**:
The user has now the chance to filter / sort events fired by the rules based on the `MITRE ATT&CK` framework classification.

```release-note
This PR includes the `MITRE ATT&CK` tags for Enterprise and for Containers. Tagging rules according to `MITRE ATT&CK` matrix is an important method of classification and will be useful for any user. The user has now the chance to filter / sort events fired by the rules based on the `MITRE ATT&CK` framework classification.
```

Signed-off-by: Alejandro Villanueva <alejandro@alejandrovillanueva.eu>
Co-authored-by: Vicente Herrera <vicenteherrera@vicenteherrera.com>
Co-authored-by: Alejandro Villanueva <alejandro@alejandrovillanueva.eu>

### CODE REVIEW

File: rules/falco_rules.yaml [hunk 1]
```diff
@@ -453,7 +453,7 @@
     a shell configuration file has been modified (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline pcmdline=%proc.pcmdline file=%fd.name container_id=%container.id image=%container.image.repository)
   priority:
     WARNING
-  tags: [file, mitre_persistence]
+  tags: [file, mitre_TA0003_persistence, mitre_TA0005_defense_evasion, mitre_T1562_impair_defenses, mitre_T1562.001_disable_or_modify_tools]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:456]: Why does it fall into the `defense evasion` category?

File: rules/falco_rules.yaml [hunk 2]
```diff
@@ -475,7 +475,7 @@
     a shell configuration file was read by a non-shell program (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline file=%fd.name container_id=%container.id image=%container.image.repository)
   priority:
     WARNING
-  tags: [file, mitre_discovery]
+  tags: [file, mitre_TA0007_discovery, mitre_TA0005_defense_evasion, mitre_T1562_impair_defenses, mitre_T1562.001_disable_or_modify_tools]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:478]: same above

File: rules/falco_rules.yaml [hunk 3]
```diff
@@ -495,7 +495,7 @@
     file=%fd.name container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority:
     NOTICE
-  tags: [file, mitre_persistence]
+  tags: [file, mitre_TA0003_persistence, mitre_TA0002_execution, mitre_TA0003_persistence, mitre_T1053_scheduled_task_job]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:498]: duplicated `mitre_TA0003_persistence`

File: rules/falco_rules.yaml [hunk 4]
```diff
@@ -870,7 +870,7 @@
     Repository files get updated (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline pcmdline=%proc.pcmdline file=%fd.name newpath=%evt.arg.newpath container_id=%container.id image=%container.image.repository)
   priority:
     NOTICE
-  tags: [filesystem, mitre_persistence]
+  tags: [filesystem, mitre_TA0003_persistence, mitre_TA0002_execution, mitre_TA0003_persistence]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:873]: duplicated `mitre_TA0003_persistence`

File: rules/falco_rules.yaml (hunk 3 above)
Comment (seishinsha) [reply rules/falco_rules.yaml]: Fixed, thanks!

File: rules/falco_rules.yaml (hunk 4 above)
Comment (seishinsha) [reply rules/falco_rules.yaml]: Fixed, thanks!

File: rules/falco_rules.yaml (hunk 2 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:478]: Reading a shell config file could lead to defense tool discovery, hence `TA0007`, and from there the next (plausible) logical step to `T1562`. Maybe the connection between these two is weak?

File: rules/falco_rules.yaml (hunk 1 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:456]: This is in connection to the next one in `Read Shell Configuration File`. By modifying these files defenses could be impaired (`T1562`) by disabling software security services.

File: rules/falco_rules.yaml (hunk 2 above)
Comment (Kaizhe) [reply rules/falco_rules.yaml:478]: but read shell config file doesn't do anything to impair defense

File: rules/falco_rules.yaml (hunk 1 above)
Comment (Kaizhe) [reply rules/falco_rules.yaml:456]: can you give me an example of modifying shell configuration leads to disabling security services?

File: rules/falco_rules.yaml [hunk 5]
```diff
@@ -2344,7 +2344,7 @@
   condition: outbound and fd.sip="169.254.169.254" and container and consider_metadata_access and not user_known_metadata_access
   output: Outbound connection to cloud instance metadata service (command=%proc.cmdline connection=%fd.name %container.info image=%container.image.repository:%container.image.tag)
   priority: NOTICE
-  tags: [network, container, mitre_discovery]
+  tags: [network, container, mitre_TA0007_discovery, mitre_TA0006_credential_access, mitre_T1552_unsecured_credentials, mitre_T1552.007_container_api]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2347]: metadata service is not container api

File: rules/falco_rules.yaml [hunk 6]
```diff
@@ -2384,7 +2384,7 @@
     not user_known_contact_k8s_api_server_activities
   output: Unexpected connection to K8s API Server from container (command=%proc.cmdline %container.info image=%container.image.repository:%container.image.tag connection=%fd.name)
   priority: NOTICE
-  tags: [network, k8s, container, mitre_discovery]
+  tags: [network, k8s, container, mitre_TA0007_discovery, mitre_TA0006_credential_access, mitre_T1552_unsecured_credentials, mitre_T1552.007_container_api]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2387]: I don't think `mitre_TA0006_credential_access` matches the rule here. Can you elaborate more?

File: rules/falco_rules.yaml [hunk 7]
```diff
@@ -2400,7 +2400,7 @@
   condition: (inbound_outbound) and fd.sport >= 30000 and fd.sport <= 32767 and container and not nodeport_containers
   output: Unexpected K8s NodePort Connection (command=%proc.cmdline connection=%fd.name container_id=%container.id image=%container.image.repository)
   priority: NOTICE
-  tags: [network, k8s, container, mitre_port_knocking]
+  tags: [network, k8s, container, mitre_TA0003_persistence, mitre_T1205.001_port_knocking]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2403]: Not a `mitre_TA0003_persistence`

File: rules/falco_rules.yaml [hunk 8]
```diff
@@ -2445,7 +2445,7 @@
     Netcat runs inside container that allows remote code execution (user=%user.name user_loginuid=%user.loginuid
     command=%proc.cmdline container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority: WARNING
-  tags: [network, process, mitre_execution]
+  tags: [network, process, mitre_TA0002_execution, mitre_TA0007_discovery, mitre_T1046_network_service_scanning]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2448]: not `mitre_T1046_network_service_scanning`

File: rules/falco_rules.yaml [hunk 9]
```diff
@@ -2458,7 +2458,7 @@
     Network tool launched in container (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline parent_process=%proc.pname
     container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority: NOTICE
-  tags: [network, process, mitre_discovery, mitre_exfiltration]
+  tags: [network, process, mitre_TA0007_discovery, mitre_TA0010_exfiltration, mitre_TA0007_discovery, mitre_T1046_network_service_scanning]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2461]: redundant `mitre_TA0007_discovery`

File: rules/falco_rules.yaml [hunk 10]
```diff
@@ -2477,7 +2477,7 @@
   output: >
     Network tool launched on host (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline parent_process=%proc.pname)
   priority: NOTICE
-  tags: [network, process, mitre_discovery, mitre_exfiltration]
+  tags: [network, process, mitre_TA0007_discovery, mitre_TA0010_exfiltration, mitre_TA0007_discovery, mitre_T1046_network_service_scanning]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2480]: redundant `mitre_TA0007_discovery`

File: rules/falco_rules.yaml (hunk 9 above)
Comment (seishinsha) [reply rules/falco_rules.yaml]: Fixed, thanks!

File: rules/falco_rules.yaml (hunk 10 above)
Comment (seishinsha) [reply rules/falco_rules.yaml]: Fixed, thanks!

File: rules/falco_rules.yaml (hunk 8 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2448]: Solved.

File: rules/falco_rules.yaml (hunk 7 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2403]: Solved

File: rules/falco_rules.yaml (hunk 5 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2347]: Solved

File: rules/falco_rules.yaml (hunk 2 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:478]: Solved

File: rules/falco_rules.yaml (hunk 1 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:456]: Solved

File: rules/falco_rules.yaml (hunk 6 above)
Comment (seishinsha) [reply rules/falco_rules.yaml]: Solved

**Review by Kaizhe — CHANGES_REQUESTED** [review]
Please address my comments accordingly

File: rules/falco_rules.yaml [hunk 11]
```diff
@@ -1999,7 +1999,7 @@
     A shell was spawned in a container with an attached terminal (user=%user.name user_loginuid=%user.loginuid %container.info
     shell=%proc.name parent=%proc.pname cmdline=%proc.cmdline terminal=%proc.tty container_id=%container.id image=%container.image.repository)
   priority: NOTICE
-  tags: [container, shell, mitre_execution]
+  tags: [container, shell, mitre_TA0002_execution, mitre_TA0002_execution, mitre_T1609_container_administration_command]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2002]: duplicated `mitre_TA0002_execution`

File: rules/falco_rules.yaml [hunk 12]
```diff
@@ -2279,7 +2279,7 @@
     User management binary command run outside of container
     (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline parent=%proc.pname gparent=%proc.aname[2] ggparent=%proc.aname[3] gggparent=%proc.aname[4])
   priority: NOTICE
-  tags: [host, users, mitre_persistence]
+  tags: [host, users, mitre_TA0003_persistence, mitre_TA0004_privilege_escalation, mitre_T1068_explotation_for_privilege_escalation]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2282]: I don't think it is related to privilege escalation. The intent of the rule is to detect an attacker tried to gain some persistence in the system (create a bogus user)

File: rules/falco_rules.yaml [hunk 13]
```diff
@@ -2303,7 +2303,7 @@
     and not user_known_create_files_below_dev_activities
   output: "File created below /dev by untrusted program (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline file=%fd.name container_id=%container.id image=%container.image.repository)"
   priority: ERROR
-  tags: [filesystem, mitre_persistence]
+  tags: [filesystem, mitre_TA0003_persistence, mitre_TA0005_defense_evasion, mitre_T1036_masquerading, mitre_T1036.005_match_legitimate_name_or_location]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2306]: I don't think it is related to `defense evasion` or `masquerading`

File: rules/falco_rules.yaml [hunk 14]
```diff
@@ -2325,7 +2325,7 @@
   condition: outbound and fd.sip="169.254.169.254" and container and not ec2_metadata_containers
   output: Outbound connection to EC2 instance metadata service (command=%proc.cmdline connection=%fd.name %container.info image=%container.image.repository:%container.image.tag)
   priority: NOTICE
-  tags: [network, aws, container, mitre_discovery]
+  tags: [network, aws, container, mitre_TA0007_discovery, mitre_TA0006_credential_access, mitre_T1552_unsecured_credentials, mitre_T1552.007_container_api]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2328]: it's not `container_api`

File: rules/falco_rules.yaml [hunk 15]
```diff
@@ -2431,7 +2431,7 @@
     Package management process launched in container (user=%user.name user_loginuid=%user.loginuid
     command=%proc.cmdline container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority: ERROR
-  tags: [process, mitre_persistence]
+  tags: [process, mitre_TA0003_persistence, mitre_TA0004_privilege_escalation, mitre_T1068_explotation_for_privilege_escalation]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2434]: not `privilege escalation`

File: rules/falco_rules.yaml [hunk 16]
```diff
@@ -2445,7 +2445,7 @@
     Netcat runs inside container that allows remote code execution (user=%user.name user_loginuid=%user.loginuid
     command=%proc.cmdline container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority: WARNING
-  tags: [network, process, mitre_execution]
+  tags: [network, process, mitre_TA0002_execution, mitre_TA0007_discovery]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2448]: this is not `discovery`

File: rules/falco_rules.yaml [hunk 17]
```diff
@@ -2515,7 +2515,7 @@
     image=%container.image.repository:%container.image.tag)
   priority:
     WARNING
-  tags: [process, mitre_credential_access]
+  tags: [process, mitre_TA0006_credential_access, mitre_TA0006_credential_access, mitre_T1552_unsecured_credentials]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2518]: duplicated `mitre_TA0006_credential_access`

File: rules/falco_rules.yaml [hunk 18]
```diff
@@ -2548,7 +2548,7 @@
     Log files were tampered (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline file=%fd.name container_id=%container.id image=%container.image.repository)
   priority:
     WARNING
-  tags: [file, mitre_defense_evasion]
+  tags: [file, mitre_TA0005_defense_evasion, mitre_TA0005_defense_evasion, mitre_T1070_indicator_removal_on_host]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2551]: duplicated `mitre_TA0005_defense_evasion`

File: rules/falco_rules.yaml [hunk 19]
```diff
@@ -2566,7 +2566,7 @@
     Bulk data has been removed from disk (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline file=%fd.name container_id=%container.id image=%container.image.repository)
   priority:
     WARNING
-  tags: [process, mitre_persistence]
+  tags: [process, mitre_TA0003_persistence, mitre_TA0005_defense_evasion, mitre_T1070_indicator_removal_on_host]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2569]: the intention of the rule is not to detect `defense evasion`

**Review by Kaizhe — CHANGES_REQUESTED** [review]

File: rules/falco_rules.yaml (hunk 11 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2002]: Removed duplicated tag.

File: rules/falco_rules.yaml (hunk 12 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2282]: I think if an unprivileged user got to execute these commands, he escalated privileges.

File: rules/falco_rules.yaml (hunk 15 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2434]: Same as above.

File: rules/falco_rules.yaml (hunk 17 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2518]: Removed duplicated tag.

File: rules/falco_rules.yaml (hunk 18 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2551]: Removed duplicated tag.

File: rules/falco_rules.yaml (hunk 19 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2569]: Replaced by `mitre_TA0040_impact` and `mitre_T1485_data_destruction`, if that's ok.

**Comment by seishinsha** [comment]
> Please address my comments properly without simply click the `resolve` button.
I honestly thought that if I resolved a comment I should click the `resolve` button. Sorry about the confusion, please bear with me. Won't be resolving comments anymore.

File: rules/falco_rules.yaml (hunk 16 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2448]: Removed `discovery` tag.

File: rules/falco_rules.yaml (hunk 14 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2328]: Removed `container_api`.

File: rules/falco_rules.yaml (hunk 13 above)
Comment (seishinsha) [reply rules/falco_rules.yaml:2306]: I think that creating files below /dev is a form of masquerading, specifically `match_legitimate_name_or_location`.

**Comment by seishinsha** [comment]
@Kaizhe, do you have any update on this PR? Thank you!

File: rules/falco_rules.yaml [hunk 20]
```diff
@@ -2566,7 +2566,7 @@
     Bulk data has been removed from disk (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline file=%fd.name container_id=%container.id image=%container.image.repository)
   priority:
     WARNING
-  tags: [process, mitre_persistence]
+  tags: [process, mitre_TA0003_persistence, mitre_TA0040_impact, mitre_T1485_data_destruction]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2569]: `persistence` seems wrong here, we should remove it.

File: rules/falco_rules.yaml [hunk 21]
```diff
@@ -2605,7 +2605,7 @@
     Shell history had been deleted or renamed (user=%user.name user_loginuid=%user.loginuid type=%evt.type command=%proc.cmdline fd.name=%fd.name name=%evt.arg.name path=%evt.arg.path oldpath=%evt.arg.oldpath %container.info)
   priority:
     WARNING
-  tags: [process, mitre_defense_evasion]
+  tags: [process, mitre_TA0005_defense_evasion, mitre_TA0005_defense_evasion, mitre_T1070_indicator_removal_on_host]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2608]: duplicated `mitre_TA0005_defense_evasion`

File: rules/falco_rules.yaml [hunk 22]
```diff
@@ -2618,7 +2618,7 @@
...
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2621]: duplicated `mitre_TA0005_defense_evasion`

File: rules/falco_rules.yaml [hunk 23]
```diff
@@ -2648,7 +2648,7 @@
     command=%proc.cmdline container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority:
     NOTICE
-  tags: [process, mitre_persistence]
+  tags: [process, mitre_TA0003_persistence, mitre_TA0004_privilege_escalation, mitre_T1068_explotation_for_privilege_escalation]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2651]: `mitre_T1068_explotation_for_privilege_escalation` probably is not the right fit here.

File: rules/falco_rules.yaml [hunk 24]
```diff
@@ -2674,7 +2674,7 @@
     file=%fd.name newpath=%evt.arg.newpath container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority:
     NOTICE
-  tags: [file, mitre_persistence]
+  tags: [file, mitre_TA0003_persistence, mitre_TA0005_defense_evasion, mitre_T1036_masquerading]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2677]: `mitre_T1036_masquerading` is not a right fit here.

File: rules/falco_rules.yaml [hunk 25]
```diff
@@ -2698,7 +2698,7 @@
     Remote file copy tool launched in container (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline parent_process=%proc.pname
     container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority: NOTICE
-  tags: [network, process, mitre_lateral_movement, mitre_exfiltration]
+  tags: [network, process, mitre_TA0008_lateral_movement, mitre_TA0010_exfiltration, mitre_TA0006_credential_access, mitre_T1552_unsecured_credentials]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2701]: the detection scenarios are not for `mitre_TA0006_credential_access` and `mitre_T1552_unsecured_credentials`

File: rules/falco_rules.yaml [hunk 26]
```diff
@@ -2708,7 +2708,7 @@
   output: >
     Symlinks created over senstivie files (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline target=%evt.arg.target linkpath=%evt.arg.linkpath parent_process=%proc.pname)
   priority: NOTICE
-  tags: [file, mitre_exfiltration]
+  tags: [file, mitre_TA0010_exfiltration, mitre_TA0005_defense_evasion, mitre_T1036_masquerading, mitre_T1036.005_match_legitimate_name_or_location]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2711]: not fit for `mitre_TA0005_defense_evasion, mitre_T1036_masquerading, mitre_T1036.005_match_legitimate_name_or_location`

File: rules/falco_rules.yaml [hunk 27]
```diff
@@ -2841,7 +2841,7 @@
   condition: spawned_process and container and not user_known_k8s_client_container_parens and proc.name in (k8s_client_binaries)
   output: "Docker or kubernetes client executed in container (user=%user.name user_loginuid=%user.loginuid %container.info parent=%proc.pname cmdline=%proc.cmdline image=%container.image.repository:%container.image.tag)"
   priority: WARNING
-  tags: [container, mitre_execution]
+  tags: [container, mitre_TA0002_execution, mitre_TA0002_execution, mitre_T1609_container_administration_command, mitre_T1610_deploy_container]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2844]: duplicated `mitre_TA0002_execution`

Comment (Kaizhe) [reply rules/falco_rules.yaml:2844]: remove `mitre_T1610_deploy_container`

File: rules/falco_rules.yaml [hunk 28]
```diff
@@ -2857,7 +2857,7 @@
   condition: evt.type=socket and evt.arg[0]=AF_PACKET and consider_packet_socket_communication and container and not proc.name in (user_known_packet_socket_binaries)
   output: Packet socket was created in a container (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline socket_info=%evt.args container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority: NOTICE
-  tags: [network, mitre_discovery]
+  tags: [network, mitre_TA0007_discovery, mitre_TA0003_persistence, mitre_T1133_external_remote_services]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2860]: remove `mitre_T1133_external_remote_services` and add `privilege eascalation`

File: rules/falco_rules.yaml (hunk 21 above)
Comment (seishinsha) [reply rules/falco_rules.yaml]: Removed duplicated tag.

File: rules/falco_rules.yaml (hunk 22 above)
Comment (seishinsha) [reply rules/falco_rules.yaml]: Removed duplicated tag.

File: rules/falco_rules.yaml (hunk 27 above)
Comment (seishinsha) [reply rules/falco_rules.yaml]: Removed duplicated tag.

**Comment by Kaizhe** [comment]
Given the changes impact almost every rules, it would be great to have a proposal with the newly added tags mapping to each rule, and what attack scenarios you have in mind. Once the proposal is there, we can schedule a time to go through all the scenarios. Communicating in github comments is not a good idea.

**Comment by seishinsha** [comment]
> Given the changes impact almost every rules, it would be great to have a proposal with the newly added tags mapping to each rule, and what attack scenarios you have in mind. Once the proposal is there, we can schedule a time to go through all the scenarios. Communicating in github comments is not a good idea.

Hey, @Kaizhe. We already put up together a spreadsheet with the proposal [in June](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-06-09), shared it and engaged the community going through it during the community calls ([1](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-06-09), [2](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-06-23), [3](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-07-14), [4](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?view#2021-07-21)) before this PR was created.

**Comment by leodido** [comment]
I second what @Kaizhe is asking.

He's asking for the proposal to be written down in [here](https://github.com/falcosecurity/falco/tree/master/proposals) so that future users/contributors can refer to it to understand the new tag mappings (changing the default rules set in this way is likely to be a BC, not in the API but in the fact that it would likely require some degree of actions by Falco users).

**Pull request closed without merging**